---
permalink: /about/
title: "About the author"
modified: 2019-09-19
comments: false
author_profile: true
header:
   #overlay_image: /images/slide-code2.png
   #overlay_filter: 0.3
---

{% include toc %}


## CV

See my [LinkedIn](https://www.linkedin.com/in/thomas-naake-2b31158a/) page for a 
current version of my CV. 

## Research

I am currently a PhD student in the group of Alisdair R. Fernie at the 
Max Planck Institute of Molecular Plant Physiology, Potsdam-Golm. My 
research projects focus on plant metabolomics and transcriptomics. 

## Publications

> See also my [Google scholar 
> profile](https://scholar.google.de/citations?user=gh9evjAAAAAJ&hl=en)
> and publications in 
> [PubMed](https://www.ncbi.nlm.nih.gov/pubmed/?term=naake+thomas).

### Research articles
Toleco M.R., Naake T., Zhang Y., Heazlewood J.L., Fernie A.R.
*Plant Mitochondrial Carriers: Molecular Gatekeepers that Help to Regulate Plant Central Carbon Metabolism*.
submitted

Perez de Souza L., Alseekh S., Naake T., Fernie A.R. 
*Mass spectrometry based untargeted plant metabolomics*. Current Protocols in Plant
Biology 4, 2019; [https://doi.org/10.1002/cppb.20100](https://doi.org/10.1002/cppb.20100).

![Metabolites Volume 9, Issue 10](/images/cover_metabolites_v9_i10.jpg){:height="50%" width="50%"}

Stanstrup J., Broeckling C.D., Helmus R., Hofmann N., Mathe E., Naake T., 
Nicolotti L., Peters K., Rainer J., Salek R.M., Schulze T., Schymanski E.L., 
Stravs M.A., Thevenot E., Treutler H., Weber R.J.M., Willighagen E., 
Witting M., Neumann S. *The metaRbolomics toolbox in Bioconductor and beyond*. 
Metabolites 9, 2019; 
[https://doi.org/10.3390/metabo9100200](https://doi.org/10.3390/metabo9100200).

Naake T., Fernie A.R. *MetNet: Metabolite Network Prediction from 
High-Resolution Mass Spectrometry Data in R Aiding Metabolite Annotation*. 
Analytical Chemistry 91, 2019; 
[https://doi.org/10.1021/acs.analchem.8b04096](https://doi.org/10.1021/acs.analchem.8b04096).

Gallage N., Jørgensen K., Janfelt C., Nielsen A., Naake, T., Dalsten L., 
Grisoni M., Møller B.L. *The vanillin biosynthetic machinery in the pods of 
Vanilla planifolia is localized in chloroplasts and phenyloplasts*. Plant Cell Physiology 59, 2018; 
[https://doi.org/10.1093/pcp/pcx185](https://doi.org/10.1093/pcp/pcx185).

Perez de Souza L., Naake T., Tohge T., Fernie A.R. *From chromatogram to 
analyte to metabolite. How to pick horses for courses from the massive 
web resources for mass spectral plant metabolomics*. Gigascience 6, 2017; 
[https://doi.org/10.1093/gigascience/gix037](https://doi.org/10.1093/gigascience/gix037).

Naake T., Gaquerel E. *MetCirc: Navigating mass spectral similarity in 
high-resolution MS/MS metabolomics data*. Bioinformatics 33, 2017; 
[https://doi.org/10.1093/bioinformatics/btx159](https://doi.org/10.1093/bioinformatics/btx159).

Christoforou A., Mulvey C.M., Breckels L.M., Geladaki A., Hurrell T., 
Hayward P.C., Naake T., Gatto L., Viner R., Arias A.M., Lilley K.S. 
*A draft map of the mouse pluripotent stem cell spatial proteome*. 
Nature Communications 7, 2016; 
[https://doi.org/10.1038/ncomms9992](https://doi.org/10.1038/ncomms9992).

Gatto L., Breckels L.M., Naake T., Gibb S. *Visualisation of proteomics data 
using R and Bioconductor*. PROTEOMICS 15, 2015;
[https://doi.org/10.1002/pmic.201400392](https://doi.org/10.1002/pmic.201400392).



### Books/book chapters 

Stanstrup J., Broeckling C.D., Helmus R., Hoffmann N., Mathe E., Naake T., 
Nicolotti L., Peters K., Rainer J., Salek R.M., Schulze T., Schymanski E.L., 
Stravs M.A., Thevenot E.A., Treutler H., Weber R.J.M., Willighagen E., 
Witting M., Neumann S. *The MetaRbolomics book*. Available at
[https://rformassspectrometry.github.io/metaRbolomics-book/](https://rformassspectrometry.github.io/metaRbolomics-book/)

Naake T., Gaquerel E., Fernie A.R. *Annotation of specialized metabolites from 
high-throughput and high-resolution mass spectrometry metabolomics*. 
Computational methods and Data Analysis for Metabolomics, Methods in Molecular 
Biology, Springer Protocols, *accepted*

## Software

I have developed open-source 
R/[Bioconductor](http://www.bioconductor.org) software packages. Check my
[GitHub page](http://www.github.com/tnaake) and in particular
the [MetCirc](http://www.github.com/tnaake/MetCirc) and 
[MetNet](http://www.github.com/tnaake/MetNet) repositories to learn more
about the packages I developed. I am also the author of the first version
[pRolocGUI](https://github.com/ComputationalProteomicsUnit/pRolocGUI)
package that was created during an ERASMUS placement at the 
Computational Proteomics Unit headed 
by [Laurent Gatto](http://lgatto.github.io) (now at UCLouvain) at the 
University of Cambridge. 

Naake T. 
*MetNet: Inferring metabolic networks from untargeted high-resolution mass spectrometry data*. 
2019; [https://doi.org/10.18129/B9.bioc.MetNet](https://doi.org/10.18129/B9.bioc.MetNet).

Naake T., Gaquerel, E. 
*MetCirc: Navigating mass spectral similarity in high-resolution MS/MS metabolomics data*. 
2019; [https://doi.org/10.18129/B9.bioc.MetCirc](https://doi.org/10.18129/B9.bioc.MetCirc)

Breckels L., Naake T., Gatto L. 
*pRolocGUI: Interactive visualisation of spatial proteomics data*. 
2019; [https://doi.org/10.18129/B9.bioc.pRolocGUI](https://doi.org/10.18129/B9.bioc.pRolocGUI)

## Recent talks 

Naake T. MetCirc: Navigating mass spectral similarity in high-resolution MS/MS metabolomics
data - Recent updates. 25/07/2019, Brixen, CSAMA: Statistical Data Analysis
for Genome-Scale Biology.

Naake T., Tohge T., Fernie A.R. Plant kingdom-wide analysis to study the evolution of the
type III polyketide synthase superfamily. 24/06/2019, Pamplona,
1st PhD meeting in Plant Science.

Naake T. MetNet: Metabolite Network Prediction from High-Resolution Mass Spectrometry
Data in R Aiding Metabolite Annotation. 02/04/2019, Wittenberg, de.NBI
CompMS metaRbolomics.


## Posters 

Naake T., Tohge T., Fernie A.R. Plant kingdom-wide analysis to study the evolution of
the polyketide synthase superfamily. 16–23/09/2018, Greifswald, Greifolution: 1st
Greifswald Summer school on Mathematics of Evolution.

Naake T., Tohge T., Fernie A.R. Exploring the natural variation of specialized seed metabolites
of Arabidopsis thaliana to detect biosynthetic and regulatory genes using GWAS.
18–21/06/2018, Copenhagen, Plant Biology Europe 2018.

Gatto L., Breckels L.M., Naake T., Wieczorek S., Burger T., Lilley K.S. A state-of-the-art
machine learning pipeline for the analysis of spatial proteomics data. 05–08/10/2014,
Madrid, HUPO meeting.

