---
permalink: /publications/
title: "Publications"
modified: 2022-07-20
comments: false
author_profile: true
header:
   #overlay_image: /images/slide-code2.png
   #overlay_filter: 0.3
---

{% include toc %}



> See also my [Google scholar 
> profile](https://scholar.google.de/citations?user=gh9evjAAAAAJ&hl=en)
> and publications in 
> [PubMed](https://www.ncbi.nlm.nih.gov/pubmed/?term=naake+thomas).

### Research articles

Naake T., Scossa F., Perez de Souza L., Borghi M., Brotman Y., Mori T., 
Nakabayashi R., Tohge T., Fernie A.R. *Genome-wide association studies of seed 
metabolites identify loci controlling specialized metabolites in Arabidopsis 
thaliana*, bioRvix, 2022;
[https://doi.org/10.1101/2022.09.23.509130](https://doi.org/10.1101/2022.09.23.509130)

Perez de Souza L., Naake T., Fernie A.R.
*Computational approaches that aid annotation in metabolomics*, 
Trends in Plant Science, 2022;
[https://doi.org/10.1016/j.tplants.2022.06.010](https://doi.org/10.1016/j.tplants.2022.06.010)

Gegner H.M., Naake T., Dugourd A., Mueller T., Czernilofsky F., Kliewer G., 
Jaeger E., Helm B., Kunze-Rohrbach N., Klingmueller U., Hopf C., 
Mueller-Tidow C., Dietrich S., Saez-Rodriguez J., Huber W., Hell R., 
Poschet G., Krijgsveld J.
*Pre-analytical processing of plasma and serum samples for combined proteome 
and metabolome analysis*, bioRxiv, 2022;
[https://doi.org/10.1101/2022.04.26.489520](https://doi.org/10.1101/2022.04.26.489520)

Amara A., Frainay C., Jourdan F., Naake T., Neumann S., Novoa-del-Toro E.M., 
Salek R., Salzer L., Scharfenberg S.,  Witting M.A.
*Networks and graphs discovery in metabolomics data analysis and interpretation*,
Frontiers in Molecular Biosciences, 2022.
[https://doi.org/10.3389/fmolb.2022.841373](https://doi.org/10.3389/fmolb.2022.841373)

Naake T., Huber W.
*MatrixQCvis: shiny-based interactive data quality exploration for omics data*,
Bioinformatics, 2022;
[https://doi.org/10.1093/bioinformatics/btab748](https://doi.org/10.1093/bioinformatics/btab748)

Zhu F., Alseekh S., Koper K., Tong H., Nikoloski Z., Naake T., Liu H., Yan J., 
Brotman Y., Wen W., Maeda H., Cheng Y., Fernie A.R.
*Genome-wide association of the metabolic shifts underpinning dark-induced 
senescence in Arabidopsis*, 
The Plant Cell, 2022;
[https://doi.org/10.1093/plcell/koab251](https://doi.org/10.1093/plcell/koab251)

Aneklaphakij C., Saigo T., Watanabe M., Naake T., Fernie A.R., Bunsupa S., Satitpatipan V., Tohge T.
*Diversity of chemical structures and biosynthesis of polyphenols in nut bearing species*,
Frontiers in Plant Science, 2021;
[https://doi.org/10.3389/fpls.2021.642581](https://doi.org/10.3389/fpls.2021.642581)

Naake T., Maeda H.A., Proost S., Tohge T., Fernie A.R.
*Kingdom-wide analysis of the evolution of the plant type III polyketide synthase superfamily*,
Plant Physiology, 2020;
[https://doi.org/10.1093/plphys/kiaa086](https://doi.org/10.1093/plphys/kiaa086).

Yan S., Liu Q., Naake T., Huang W., Chen M., Kong Q., Zhang S., Li W., Li X., Liu Q., Yang J., Fernie A.R., Liu B.
*OsGF14b modulates defense signaling pathways in rice panicle blast response*. The Crop Journal, 2020;
[https://doi.org/10.1016/j.cj.2020.10.007](https://doi.org/10.1016/j.cj.2020.10.007).

Aarabi F., Naake T., Fernie A.R., Hoefgen R. 
*Coordinating Sulfur Pools under Sulfate Deprivation*. Trends in Plant Science, 2020;
[https://doi.org/10.1016/j.tplants.2020.07.007](https://doi.org/10.1016/j.tplants.2020.07.007).

Toleco M.R., Naake T., Zhang Y., Heazlewood J.L., Fernie A.R.
*Plant Mitochondrial Carriers: Molecular Gatekeepers that Help to Regulate Plant Central Carbon Metabolism*. 
Plants 9, 2020; 
[https://doi.org/10.3390/plants9010117](https://doi.org/10.3390/plants9010117).

Perez de Souza L., Alseekh S., Naake T., Fernie A.R. 
*Mass spectrometry based untargeted plant metabolomics*. Current Protocols in Plant
Biology 4, 2019; [https://doi.org/10.1002/cppb.20100](https://doi.org/10.1002/cppb.20100).

![Metabolites Volume 9, Issue 10](/images/cover_metabolites_v9_i10.jpg){:height="50%" width="50%"}

Stanstrup J., Broeckling C.D., Helmus R., Hofmann N., Mathe E., Naake T., 
Nicolotti L., Peters K., Rainer J., Salek R.M., Schulze T., Schymanski E.L., 
Stravs M.A., Thevenot E., Treutler H., Weber R.J.M., Willighagen E., 
Witting M., Neumann S. *The metaRbolomics toolbox in Bioconductor and beyond*. 
Metabolites 9, 2019; 
[https://doi.org/10.3390/metabo9100200](https://doi.org/10.3390/metabo9100200).

Naake T., Fernie A.R. *MetNet: Metabolite Network Prediction from 
High-Resolution Mass Spectrometry Data in R Aiding Metabolite Annotation*. 
Analytical Chemistry 91, 2019; 
[https://doi.org/10.1021/acs.analchem.8b04096](https://doi.org/10.1021/acs.analchem.8b04096).

Gallage N., Jørgensen K., Janfelt C., Nielsen A., Naake, T., Dalsten L., 
Grisoni M., Møller B.L. *The vanillin biosynthetic machinery in the pods of 
Vanilla planifolia is localized in chloroplasts and phenyloplasts*. Plant Cell Physiology 59, 2018; 
[https://doi.org/10.1093/pcp/pcx185](https://doi.org/10.1093/pcp/pcx185).

Perez de Souza L., Naake T., Tohge T., Fernie A.R. *From chromatogram to 
analyte to metabolite. How to pick horses for courses from the massive 
web resources for mass spectral plant metabolomics*. Gigascience 6, 2017; 
[https://doi.org/10.1093/gigascience/gix037](https://doi.org/10.1093/gigascience/gix037).

Naake T., Gaquerel E. *MetCirc: Navigating mass spectral similarity in 
high-resolution MS/MS metabolomics data*. Bioinformatics 33, 2017; 
[https://doi.org/10.1093/bioinformatics/btx159](https://doi.org/10.1093/bioinformatics/btx159).

Christoforou A., Mulvey C.M., Breckels L.M., Geladaki A., Hurrell T., 
Hayward P.C., Naake T., Gatto L., Viner R., Arias A.M., Lilley K.S. 
*A draft map of the mouse pluripotent stem cell spatial proteome*. 
Nature Communications 7, 2016; 
[https://doi.org/10.1038/ncomms9992](https://doi.org/10.1038/ncomms9992).

Gatto L., Breckels L.M., Naake T., Gibb S. *Visualisation of proteomics data 
using R and Bioconductor*. PROTEOMICS 15, 2015;
[https://doi.org/10.1002/pmic.201400392](https://doi.org/10.1002/pmic.201400392).



### Books/book chapters 

Stanstrup J., Broeckling C.D., Helmus R., Hoffmann N., Mathe E., Naake T., 
Nicolotti L., Peters K., Rainer J., Salek R.M., Schulze T., Schymanski E.L., 
Stravs M.A., Thevenot E.A., Treutler H., Weber R.J.M., Willighagen E., 
Witting M., Neumann S. *The MetaRbolomics book*. Available at
[https://rformassspectrometry.github.io/metaRbolomics-book/](https://rformassspectrometry.github.io/metaRbolomics-book/).

Naake T., Gaquerel E., Fernie A.R. *Annotation of specialized metabolites from 
high-throughput and high-resolution mass spectrometry metabolomics*. 
Springer Protocols, Methods in Molecular Biology, Computational methods and 
Data Analysis for Metabolomics, 2020; 
[https://doi.org/10.1007/978-1-0716-0239-3](https://doi.org/10.1007/978-1-0716-0239-3).



### Recent talks 

Neumann S., Burgess K., da Silva R.R., Schirra J., Naake T. 
Hitchhikers’ guide to networks in metabolomics. 20/06/2022, Valencia, Metabolomics 2022.

Naake T. MsQuality: quality metrics for mass spectrometry data. 
23/11/2021, Wittenberg, de.NBI CompMS metaRbolomics.

Naake T. MetCirc: Navigating mass spectral similarity in high-resolution MS/MS metabolomics
data - Recent updates. 25/07/2019, Brixen, CSAMA: Statistical Data Analysis
for Genome-Scale Biology.

Naake T., Tohge T., Fernie A.R. Plant kingdom-wide analysis to study the evolution of the
type III polyketide synthase superfamily. 24/06/2019, Pamplona,
1st PhD meeting in Plant Science.

Naake T. MetNet: Metabolite Network Prediction from High-Resolution Mass Spectrometry
Data in R Aiding Metabolite Annotation. 02/04/2019, Wittenberg, de.NBI
CompMS metaRbolomics.


### Posters 

Naake T., Rainer J., Huber W. MsQuality - an interoperable open-source package for the
calculation of standardized quality metrics of mass
spectrometry data. 19-23/06/2022, Valencia, Metabolomics 2022.

Naake T., Huber W. MatrixQCvis - shiny-based interactive data quality exploration 
for omics data. 27-30/09/2021, Bad Dürkheim, MsCoreSys Summer School: 
Mass spectrometry meets system medicine.

Naake T., Tohge T., Fernie A.R. Plant kingdom-wide analysis to study the evolution of
the polyketide synthase superfamily. 16–23/09/2018, Greifswald, Greifolution: 1st
Greifswald Summer school on Mathematics of Evolution.

Naake T., Tohge T., Fernie A.R. Exploring the natural variation of specialized seed metabolites
of Arabidopsis thaliana to detect biosynthetic and regulatory genes using GWAS.
18–21/06/2018, Copenhagen, Plant Biology Europe 2018.

Gatto L., Breckels L.M., Naake T., Wieczorek S., Burger T., Lilley K.S. A state-of-the-art
machine learning pipeline for the analysis of spatial proteomics data. 05–08/10/2014,
Madrid, HUPO meeting.

